ansible/playbooks/AdoptOpenJDK_Linux_Playbook/roles/Common/vars/Ubuntu.yml

```diff
@@ -61,5 +61,6 @@ Test_Tool_Packages:
   - pulseaudio
   - xauth
   - xorg
+  - mercurial
```

Comment:
Could we put this in alphabetical order please?

Reply:
Done!

Just a query, why is it placed under the `Test_Tool_Packages` category, its a version control system, more like to fall under an appropriate source code / build tools category if we want to be correct about categorisation.

Reply:
That's fair. Its up to you if you'd like to move it there.